tests/lib/config.js

```diff
@@ -28,6 +28,9 @@ var userAgent = {
 // set default viewport size to desktop
 casper.options.viewportSize = viewport.desktop;
 
+// set default wait timeout
+casper.options.waitTimeout = 5000;
```

Comment:
Is resetting it in helpers#done not enough? The default timeout in Casperjs is 5000